portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java

```diff
@@ -33,17 +33,21 @@
   private static final int COUNTRY_INDEX = 1;
   private static final int RICE_INDEX = 4;
 
-  // Stores data in the form "country_name: rice_produced."
+  // Stores data in the form "country_name: rice_supplies."
```

Comment:
this this country_name here csv_country_name or geochart_country_name?

Reply:
This should be geochart_country_name -- I'll specify in the comment.